### PR TITLE
chore: adjust wallet-button styling + add logo-href 

### PR DESCRIPTION
**Pull request by dennyscode**
Ticket: https://lifi.atlassian.net/browse/LF-10487

### CODE REVIEW

**Review by oktapodia — APPROVED** [review]
LGTM

**Comment by tcheee** [comment]
image
I still have an issue when my wallet is not connected

File: src/components/Navbar/WalletButton.style.ts [hunk 1]
```diff
@@ -78,3 +93,13 @@ export const SkeletonWalletMenuButton = styled(Skeleton)(({ theme }) => ({
   marginRight: 1.1,
   marginLeft: 1.1,
 }));
+
+export const WalletLabel = styled(Typography)(({ theme }) => ({
+  display: 'none',
+  [theme.breakpoints.up('md')]: {
```
Comment (tcheee) [inline src/components/Navbar/WalletButton.style.ts:99]: what about when this the button connect

**Comment by dennyscode** [comment]
@tcheee What do you think about adding a tooltip for more context of those new icons?
Screenshot 2024-10-17 at 16 20 48

File: src/components/Portfolio/Portfolio.styles.ts [hunk 2]
```diff
@@ -105,9 +101,7 @@ export const PortfolioBox = styled(Box)(({ theme }) => ({
   flexDirection: 'column',
   display: 'flex',
   justifyContent: 'center',
-  padding: '16px',
-  paddingTop: '0px',
-  paddingBottom: '0px',
+  padding: theme.spacing(0, 2),
```
Comment (dennyscode) [inline src/components/Portfolio/Portfolio.styles.ts:104]: @tcheee Tested it, seemed to work just fine for me here

**Review by tcheee — APPROVED** [review]

**Pull request merged**